utils/btrfs-progs/Makefile

```diff
@@ -90,7 +91,8 @@ define Package/btrfs-progs/install
 	$(foreach prog,$(boxprogs),$(LN) btrfs $(1)/usr/bin/$(prog);)
 	$(foreach prog,$(progs),$(INSTALL_BIN) $(PKG_BUILD_DIR)/$(prog) $(1)/usr/bin/;)
 	$(INSTALL_DIR) $(1)/lib/preinit
-	$(INSTALL_BIN) ./files/btrfs-scan.init $(1)/lib/preinit/85_btrfs_scan
+	$(INSTALL_BIN) ./files/btrfs-scan.init $(1)/lib/preinit/85_btrfs_scan_post
+	ln -srfv $(1)/lib/preinit/85_btrfs_scan_post $(1)/lib/preinit/78_btrfs_scan_pre
```

Comment:
$(LN) not enough here?

Reply:
I don't know. Is **$(LN)** documented somewhere? What does it do?

Reply:
https://github.com/openwrt/openwrt/blob/main/rules.mk#L308

Reply:
Ah, ok, it won't work. Without -r it will link to the absolute path in the build dir tree.

Reply:
Looks like you can avoid by doing ./

Reply:
Probably relevant: https://github.com/openwrt/packages/pull/25340

Reply:
I pushed the requested modification, but I didn't test it.